core/templates/dev/head/pages/exploration_editor/simple_editor_tab/StatesToQuestionsService.js

```diff
@@ -174,6 +174,13 @@ oppia.factory('StatesToQuestionsService', [
       // current set of states to a list of questions.
       getQuestions: function() {
         return getQuestions();
+      },
+      getHumanReadableQuestionType: function(interactionId) {
+        for (key in SUPPORTED_INTERACTION_TYPES) {
```

Comment:
That said, SUPPORTED_INTERACTION_TYPES is a list. Shouldn't you be using "for var i = 0; i < ....length; i++)"?